Makefile.doc

```diff
@@ -128,7 +128,7 @@ sphinx-clean:
 plugin-tutorial: states $(BCONTEXT)/coq-core.install | $(VO_OUT_DIR)
 	mkdir -p $(VO_OUT_DIR)/tools/
 	cp tools/CoqMakefile.in $(VO_OUT_DIR)/tools/
-	+$(MAKE) COQBIN=$(shell pwd)/$(CBIN)/ COQCORELIB=$(shell pwd)/$(CONTEXT)/lib/coq-core COQLIB=$(shell pwd)/$(VO_OUT_DIR) -C $(PLUGINTUTO)
+	+$(MAKE) OCAMLPATH=$(shell pwd)/$(CONTEXT)/lib/ COQBIN=$(shell pwd)/$(CBIN)/ COQCORELIB=$(shell pwd)/$(CONTEXT)/lib/coq-core COQLIB=$(shell pwd)/$(VO_OUT_DIR) -C $(PLUGINTUTO)
```

Comment:
What's wrong with the OCAMLPATH exported in Makefile.common?

Reply:
Not sure, but I seem to recall that `export` is for "subshells", while here we call make.
Maybe I was mistaken, but at some point the sub make was not seeing the updated OCAMLPATH.

Reply:
>Not sure, but I seem to recall that export is for "subshells", while here we call make.

that is incorrect
export means put in environment 
`foo=bla bli` means put in environment just for this command